.github/workflows/test.yml

```
@@ -0,0 +1,44 @@
+name: fontsan CI
```

I'm not sure it's necessary to repeat `fontsan` here.

```suggestion
name: CI
```